source/server/CMakeLists.txt

```diff
@@ -90,7 +88,7 @@ else()
 endif()
 
 add_executable(${QFUSION_SERVER_NAME} ${SERVER_BINARY_TYPE} ${SERVER_HEADERS} ${SERVER_SOURCES} ${SERVER_PLATFORM_SOURCES})
-add_dependencies(${QFUSION_SERVER_NAME} angelwrap game)
+add_dependencies(${QFUSION_SERVER_NAME} angelwrap game wf_steam)
```

Comment:
If `BUILD_STEAMLIB` is not set, `wf_steam` will not exist.
Either add it as an `interface` library (or a fake target),
or fill a variable with `wf_steam` if `BUILD_STEAMLIB` is set, and nothing when `BUILD_STEAMLIB` is not set, and use that variable here.